### ISSUE TITLE
[Bug]: steps component

### ISSUE THREAD

**Issue by franklinogf** [robsontenorio/mary#295]
### MaryUI version

1.20

### daisyUI version

4.6.2

### Livewire version

3.4

### What browsers are affected?

Chrome

### What happened?

# The Timeline component icons are not well positioned
![image](https://github.com/robsontenorio/mary/assets/8560372/165eb962-47ac-4ecb-832b-44b06d1b0f92)

**Comment by robsontenorio** [comment]
You have sent a image from maryUI docs. There we use daisyUI 4.6.0

Have you tried to run locally the example ?

**Comment by franklinogf** [comment]
Yes, I'm using Chrome, and i used it locally on my project and it look the same, and I updated Mary and Daisy to the last version 

![image](https://github.com/robsontenorio/mary/assets/8560372/83bb7d96-a9ad-4956-9678-f9c80b0d8af1)

**Comment by robsontenorio** [comment]
Is this a brand new project ? Or do you have something extra on your `app.css` ?

**Comment by franklinogf** [comment]
it is a brand new project <code>app.css</code> only have the tailwindcss directives
